Fall-2015/HW-5/p1/p1.md

```diff
@@ -0,0 +1,23 @@
+سری اعداد زير را در نظر بگيريد.
+$$
+1, 3, 6, 10, 15
+$$
+در اين سري جمله اول برابر 1 است و جمله n-ام برابر با جمع "مقدار جمله n-1" و "عدد n" است.
+الف) تابعی بنويسيد كه عدد n را به عنوان ورودی بگيرد و مقدار جمله n-ام را به عنوان نتيجه توليد كند (برگرداند)  
+ب) تابعی بنويسيد كه عدد x را بگيرد، اگر x يك جمله در اين سری باشد شماره جمله آن به عنوان نتيجه توليد شود و در غير اين صورت عدد 1- برگردانده شود. مثلا اگر x برابر 10 باشد، نتيجه برابر 4 خواهد بود.  
+ج) برنامه‌ای بنويسيد كه دو عدد a و b را از كاربر بگيرد و همه اعداد در بازه (a, b] را كه عضوی از اين سری هستند را با شماره جمله مشخص كند.  
+ تذکر:  
+ * بدیهی است، آنچه شما برای این سوال آپلود می‌کنید، پاسخ قسمت (ج) می‌باشد.
+
+
+ مثال:  
+
+input:
+
```

Comment:
These should be in a code block with the backtick.

Reply:
Thanks for pointing this out . 
Has been Applied to all cases .